### DISCUSSION TITLE
What is the parity between LlamaIndex and LlamaIndexTS?

### DISCUSSION THREAD

**Discussion by trevorpfiz** [run-llama/LlamaIndexTS#674] [Q&A] [answered]
Hi!

How similar are the Python and Typescript versions of LlamaIndex, and what can or can't you do in each?

I am starting a new project using Next.js, and I am wondering what the best backend strategy will be. I assume the best is a hybrid approach of using Next.js with LlamaIndex.TS for most things and then using a separate FastAPI with the Python version of LlamaIndex when needed, but if LlamaIndex.TS is quite limited compared to the Python version I would just start on FastAPI.

Thanks!

**Comment by marcusschiesser** [comment, marked as answer]
LlamaIndexTS aims to support all core features of the Python version.

So unless you directly missing a feature in LlamaIndexTS and assuming that you want to benefit from Next.js features like server actions, I would start with LlamaIndexTS.

**Reply by balasphilip** [reply]
Hello, don't see any capabilities to work with Property Graphs, unfortunately
